projects/rocprofiler-sdk/source/lib/python/rocpd/summary.py

```diff
@@ -209,20 +227,27 @@ def generate_domain_query(connection: RocpdImportData, by_rank=False) -> Tuple[s
         join_condition = "CROSS JOIN total_duration TD"
         order_by = 'ORDER BY GD."DURATION (nsec)" DESC'
 
-    summary_views = [
-        itr for itr in get_temp_view_names(connection) if itr.endswith(view_suffix)
-    ]
+    summary_views = {
```

Comment:
```suggestion
    summary_dictionary = {
```
or some other better name, now that they're not views (and you used summary_queries already :D)

Reply:
Done.